src/com/facebook/buck/jvm/groovy/GroovycStep.java

```diff
@@ -91,14 +98,61 @@ public String getDescription(ExecutionContext context) {
   }
 
   private ImmutableList<String> createCommand() {
-    ImmutableList.Builder<String> command = ImmutableList.builder();
-    command.addAll(groovyc.getCommandPrefix(resolver))
+    final ImmutableList.Builder<String> command = ImmutableList.builder();
+
+    command.addAll(groovyc.getCommandPrefix(resolver));
+
+    String classpath = Joiner.on(":").join(transform(declaredClasspathEntries, toStringFunction()));
```

Comment:
Add a lint if that's the case? Not sure how external people would know and this is perfect for lint.